<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>list_selector</class>
 <widget class="QWidget" name="list_selector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>331</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLineEdit" name="filter_line_edit"/>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="filter_type_combo_box">
     <item>
      <property name="text">
       <string>Include</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Exclude</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QCheckBox" name="show_selected_checkbox">
     <property name="text">
      <string>Show Selected</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <layout class="QGridLayout" name="list_layout"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="number_of_selected_display_box">
     <property name="text">
      <string>Displaying 0 of 0 selected</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="unselect_all_button">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Unselect All</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="select_all_button">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Select All</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>filter_line_edit</tabstop>
  <tabstop>filter_type_combo_box</tabstop>
  <tabstop>show_selected_checkbox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
